x-pack/test/functional/apps/lens/index.ts

```diff
@@ -5,84 +5,122 @@
  * 2.0.
  */
 
+import { EsArchiver } from '@kbn/es-archiver';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getService, loadTestFile, getPageObjects }: FtrProviderContext) {
+export default ({ getService, loadTestFile, getPageObjects }: FtrProviderContext) => {
   const browser = getService('browser');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
   const PageObjects = getPageObjects(['timePicker']);
+  const config = getService('config');
+  let remoteEsArchiver;
 
   describe('lens app', () => {
+    const localIndexPatternString = 'logstash-*';
+    const remoteIndexPatternString = 'ftr-remote:logstash-*';
+    const localFixtures = {
+      esArchive: 'x-pack/test/functional/es_archives/logstash_functional',
+      lensBasic: 'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json',
+      lensDefault: 'x-pack/test/functional/fixtures/kbn_archiver/lens/default',
+    };
+
+    const remoteFixtures = {
+      esArchive: 'x-pack/test/functional/es_archives/logstash_functional',
+      lensBasic: 'x-pack/test/functional/fixtures/kbn_archiver/lens/ccs/lens_basic.json',
+      lensDefault: 'x-pack/test/functional/fixtures/kbn_archiver/lens/ccs/default',
+    };
+    let esNode: EsArchiver;
+    let fixtureDirs: {
+      esArchive: string;
+      lensBasic: string;
+      lensDefault: string;
+    };
+    let indexPatternString: string;
     before(async () => {
-      log.debug('Starting lens before method');
+      await log.debug('Starting lens before method');
       await browser.setWindowSize(1280, 1200);
-      await esArchiver.load('x-pack/test/functional/es_archives/logstash_functional');
+      try {
+        config.get('esTestCluster.ccs');
```

Comment:
It wasn't clear to me at first, but I think this `config.get('esTestCluster.ccs');` is essentially the `if (ccs),` and the `catch` is the `else`?

Is there an easy way to use something like (I think it would be more obvious);
`if (config.keys().includes('esTestCluster.ccs')) {` ?

Reply:
or just `if (config.get('esTestCluster.ccs')) {` like is used on line 75

Reply:
Oh shoot. Yeah. I had that block there when debugging. From another example in `test/common/services/security/system_indices_user.ts` and it's `if (config.get('esTestCluster.ccs'))` which @spalger put there. So we should probably follow that way. I was using the try catch to debug when there were some errors happening.